src/main/java/frc/robot/Feedback/LED/Limelight.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.Feedback.LED;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class Limelight extends CommandBase {
+  /** Creates a new Limelight. */
+  private TowerSubsystem m_tower;
+  public Limelight(TowerSubsystem m_tower) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    public static NetworkTableInstance table = NetworkTableInstance.getDefault();
+    this.m_tower = m_tower;
+    addRequirements(m_tower);
```

Comment:
This command will stop any currently running Tower operations (like moving the rollers) since it `requires()` the tower, so you can remove the requirements here.